Source/UI/Mac/ECOptionsMenu.m

```diff
@@ -80,7 +80,12 @@ - (void)buildMenuWithOptions:(NSDictionary*)options action:(SEL)action
 
 		NSMenuItem* item = [[NSMenuItem alloc] initWithTitle:title action:action keyEquivalent:@""];
 		item.target = self;
-		item.representedObject = option;
+		NSObject *value = optionData[@"value"];
+		if (value != nil) {
+			item.representedObject = value;
+		} else {
+			item.representedObject = option;
+		}
```

Comment:
Maybe a ternary operator would be nicer here?

```
item.representedObject = (value ? value : option)
```

Reply:
agreed, will change that!